tests/sql/parsing/test_ast.py

```diff
@@ -497,9 +500,9 @@ def test_map_subscripts():
     assert compare_query_strings('SELECT some_map["x"] FROM a', str(query))
 
 
-def test_query_to_select(cte_query):
+def test_queryto_select(cte_query):
```

Comment:
Looks like collateral damage from a `codemod "_to_select" "to_select"` ?

Reply:
You're exactly right, that was the exact find and replace that I did. 😅 (fixed it!)